cicd/gitops/understanding-openshift-gitops.adoc

```diff
@@ -0,0 +1,12 @@
+[id="understanding-openshift-gitops"]
+= Understanding OpenShift GitOps
```

Comment:
I see there's sort of three different names going on here, between "OpenShift GitOps", plain ol' "GitOps", and "Red Hat OpenShift GitOps" (which I see is set as `:gitops-title:`).

If the "About GitOps" section is intended to just discuss the general purpose of GitOps overall (not specific to the RH product), maybe re-title that one as something like "About GitOps principles" or "About GitOps methodology"?

And if "OpenShift GitOps" is not an approved shortened name (I see `:gitops-shortname:` is just set to `GitOps` below), probably should use "Red Hat OpenShift GitOps" in the title here and the left-nav (in `_topic_map.yml`).